### PR TITLE
Task: Update .

### PR DESCRIPTION
**Pull request by nspragg**
Updating the project to use const, lambda and linting rules.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Support for graphite-style relative times in Moment",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/bbc/moment-relative.git"
   },
+  "engines": {
+    "node": ">4.0.0"
```
Comment (cejast) [inline package.json]: Think you meant this:

```suggestion
    "node": ">=4.0.0"
```

File: package.json [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+  },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "eslint .",
+    "posttest": "npm run lint",
+    "coverage": "istanbul cover _mocha -- -R dot"
```
Comment (cejast) [inline package.json]: While we're at it, it might be nice to change this to use `nyc`. Another PR, perhaps.

Comment (nspragg) [reply package.json]: Yes, we can address that separately

File: package.json (hunk 1 above)
Comment (nspragg) [reply package.json]: Indeed. Let's move to 6.

**Comment by cejast** [comment]
:+1:

**Pull request merged**